rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -65,6 +67,8 @@ install -m 644 Makefile %{buildroot}/usr/share/%{name}/Makefile
 install -m 755 scripts/* %{buildroot}/usr/share/%{name}/scripts/
 install -m 644 launcher/*.py %{buildroot}/opt/securedrop/launcher/
 install -m 644 launcher/sdw_updater_gui/*.py %{buildroot}/opt/securedrop/launcher/sdw_updater_gui/
+install -m 644 launcher/sdw_notify/*.py %{buildroot}/opt/securedrop/launcher/sdw_notify/
+install -m 644 launcher/sdw_util/*.py %{buildroot}/opt/securedrop/launcher/sdw_util/
```

Comment:
Note we're currently not marking `sdw-notify.py` or `sdw-launcher.py` as executable here; that's enforced in the dom0 Salt state.

Reply:
Done, but that required changing the wildcard pattern under `%files` (to avoid duplication warnings), which in turn exposed the issue described by @conorsch in #449. We'll need to decide whether to explicitly include the `*.pyc`/`*.pyo` files (previously they were _implicitly_ included by wildcard), explicitly _exclude_ them via `%exclude`, or disable the build root policy that generates them in the first place; see discussion in #449.